tapdance/logger.go

```diff
@@ -19,14 +19,17 @@ var initLoggerOnce sync.Once
 
 // Logger is an access point for TapDance-wide logger
 func Logger() *logrus.Logger {
-	const build string = "[BUILD]"
-
 	initLoggerOnce.Do(func() {
 		logrusLogger = logrus.New()
 		logrusLogger.Formatter = new(formatter)
 		logrusLogger.Level = logrus.InfoLevel
-		//logrusLogger.Level = logrus.DebugLevel
-		logrusLogger.Infof("Running gotapdance build %s", build)
+		// logrusLogger.Level = logrus.DebugLevel
+
+		// CI will overwrite the build string with branch and commit hash
+		const build string = "[BUILD]"
```

Comment:
Does it actually get overwritten? My cli prints out "Running gotapdance build [BUILD]". Is there a script we're supposed to run to replace this?

Reply:
Is the comment above the line you just commented ambiguous, or you haven't noticed it?